src/cmd-upload-oscontainer

```diff
@@ -58,6 +59,20 @@ if not os.path.exists(tmprepo):
                            f'{latest_build_path}/{ostree_commit_tar}',
                            '-C', tmprepo])
 
+tmp_osreleasedir = 'tmp/usrlib-osrelease'
+cmdlib.run_verbose(['/usr/bin/ostree', 'checkout', '--repo', tmprepo,
+                    '--user-mode', '--subpath=/usr/lib/os-release', ostree_commit,
+                    tmp_osreleasedir])
+display_name = None
+with open(os.path.join(tmp_osreleasedir, "os-release")) as f:
+    for line in f.readlines():
+        if not line.startswith('NAME='):
+            continue
+        display_name = line.split('=', 2)[1]
```

Comment:
Why 2 maxsplits? Does RHCOS have a trailing `=`?

Reply:
No, but it's just a general best practice when parsing.